src/arch/svm/eventsClient.ts

```diff
@@ -70,14 +82,14 @@ export class SvmSpokeEventsClient {
    * @param options - Options for fetching signatures.
    * @returns A promise that resolves to an array of events matching the eventName.
    */
-  public async queryEvents<T extends EventData>(
-    eventName: EventName,
+  public async queryEvents(
+    eventName: string,
     fromBlock?: bigint,
     toBlock?: bigint,
     options: GetSignaturesForAddressConfig = { limit: 1000, commitment: "confirmed" }
-  ): Promise<EventWithData<T>[]> {
+  ): Promise<EventWithData[]> {
     const events = await this.queryAllEvents(fromBlock, toBlock, options);
-    return events.filter((event) => event.name === eventName) as EventWithData<T>[];
+    return events.filter((event) => event.name.toLowerCase() === eventName.toLowerCase()) as EventWithData[];
```

Comment:
Do we want to permit events to be specified in the wrong casing on the `queryEvents` call? On the EVM side we require that they're exact, but here we normalise via `.toLowerCase()`. I'd think we could skip the normalisation. wdyt?

Reply:
Yeah you're probably right. https://github.com/across-protocol/sdk/pull/982/commits/466f977e29ce3a6412595be11dbdce06945e41b7